packages/gatsby-plugin-postcss/README.md

```diff
@@ -1,6 +1,6 @@
 # gatsby-plugin-postcss
 
-Gatsby plugin to handle PostCSS.
+Provides drop-in support for PostCSS
```

Comment:
I'm unsure what you mean with this? :)

Reply:
I aligned the heading with the rest of the styling plugins